Makefile

```diff
@@ -75,7 +75,7 @@ CXXFLAGS += $(ALSA_CFLAGS)
 LDFLAGS += $(ALSA_LDFLAGS)
 endif
 
-CORE_SRCS = ./src/base64_encode.cc ./src/json_util.cc
+CORE_SRCS = ./src/base64_encode.cc
```

Comment:
Is there a reason to have removed the json_util file? Won't that prevent the project from compiling?

Reply:
I've reworked this PR without this . I'd address that in another PR.